charters/DataPortal/charter.md

```diff
@@ -0,0 +1,80 @@
+* DevSecOps compliance as required by the HCA Security Team
+* Leverage the authentication and authorization implementation libraries from the DevSecOps team to implement login and access control for the Data Portal and Data Browser
+* Community engagement
+   * Triage and integration of feature requests into the work queue 
+   * Support plan for escalation of user reported bugs
+### Data Portal
+* Select and deploy a content management system
+* Integrate Data Browser and Data Portal UIs into a seamless UX and provide link to other groups' applications (e.g. the Data Ingestion UI)
+* Frequent updates of summary statistics (cells, organs, donors, projects, labs, etc.) on the Home Page
+* Coordinate development of user guide content (e.g. “Learn”) from multiple contributors
+* Coordinate development of developer documentation content (e.g. “Develop”) from multiple contributors. Includes API documentation and development guides.
+* Click through to Data Browser from search box and human body image
+* Registration form for tertiary portals and list of portals available with click through to hosting web site (e.g. Dockstore)
+* Integration with Zendesk help desk
+* Integration with Vanilla Forums (discussion forum for developers)
+* Integration with Canny (to capture user feedback)
+### Data Browser
+* Create (or use Data Storage Service [DSS] provided) indexes to access files in the Data Store by using public DSS APIs to read the metadata
+* Browser support for multiple metadata schemas that are supported by the schema access library maintained by the Ingest service
+* Search views of Specimens, Projects and Files
+* Support for specifying multiple search criteria (“facets”) through the Browser UI
+* Download selected files from the Data Store via the UI and/or HCA DCP Command Line Interface (CLI)
+* Browser interface to the Date Store Collections service to save, view, list, share and delete the query results from a search operation
+* Search results / Collections handoff to a tertiary portal. Provide for direct handoff of search results to FireCloud.
+* Facilities to save, view, share, update, and delete search queries and personal Collections on a per-user basis
+* Search results / Collections handoff to the Matrix service
+* API Documentation for portable, cloud-neutral APIs
+
+## Out-of-scope
+* Production operation of applications and services linked to from the Data Portal (e.g. the DSS, data ingestion service, Secondary Analysis workflows, Tertiary Analysis portals, etc.)
+* Any separate data ingest application developed by The European Bioinformatics Institute (EBI) which may replace or augment the "Contribute" tab in the Data Portal
+* Methods Registry providing management, searching, filtering, sorting, visualization, and methods packages in the tertiary portal (registry is developed and maintained by the Broad Institute)
+* Primary interface for collecting user feedback and feature/bug requests
```

Comment:
This seems like a bad thing to be out of scope for the data portal. 

The data portal will be the first portal of call for most users of the HCA for the DCP which means it makes most sense for it to be the primary location for leaving feedback. I know @gabsie and @adriennes have thought about this